po/de.po

```diff
@@ -0,0 +1,455 @@
+# Martin Pitt <mpitt@redhat.com>, 2019. #zanata
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE_VERSION\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-30 08:05+0000\n"
+"PO-Revision-Date: 2019-10-29 08:13+0000\n"
+"Last-Translator: Martin Pitt <mpitt@redhat.com>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"X-Generator: Zanata 4.6.2\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+
+#: src/ImageDeleteModal.jsx:16
+msgid "Are you sure you want to delete this image?"
+msgstr "Sind Sie sicher, dass Sie dieses Image löschen wollen?"
+
+#: src/ContainerCommitModal.jsx:171 src/ImageDetails.jsx:28
+msgid "Author"
+msgstr "Autor"
+
+#: src/app.jsx:401
+msgid "Automatically start podman on boot"
+msgstr "podman automatisch beim Hochfahren starten"
+
+#: src/Containers.jsx:212 src/ImageUsedBy.jsx:40
+msgid "CPU"
+msgstr "CPU"
+
+#: src/ContainerCommitModal.jsx:214 src/ContainerRemoveErrorModal.jsx:19
+#: src/ImageRemoveErrorModal.jsx:19 src/ImageRunModal.jsx:379
+#: src/ImageSearchModal.jsx:184
+msgid "Cancel"
+msgstr "Abbruch"
+
+#: src/ContainerCommitModal.jsx:181 src/ContainerDetails.jsx:35
+#: src/Containers.jsx:212 src/ImageDetails.jsx:24 src/ImageRunModal.jsx:294
+#: src/ImageRunModal.jsx:298 src/ImageUsedBy.jsx:40
+msgid "Command"
+msgstr "Befehl"
+
+#: src/ContainerCommitModal.jsx:215 src/Containers.jsx:137
+msgid "Commit"
+msgstr ""
+
+#: src/ContainerCommitModal.jsx:206
+msgid "Commit Image"
+msgstr ""
+
```

Comment:
heh, I also had no idea what would be correct translation of `commit`

Reply:
In non-computer language it would be something like "verpflichten", but this sounds absurd in the context of containers. When German geeks talk to each other about database transactions, git revisions, etc, they literally say "etwas committen", because there just isn't any good German word for it that doesn't sound ridiculous.